src/KeyboardAccessoryNavigation.js

```diff
@@ -148,8 +172,11 @@ KeyboardAccessoryNavigation.defaultProps = {
   previousDisabled: false,
   nextHidden: false,
   previousHidden: false,
+  doneButtonHitslop: { left: 0, top: 0, right: 0, bottom: 0 },
```

Comment:
Can we keep the defaults as undefined.

Reply:
Yes, for now, but I deleted the part because I thought not setting the default would not cause any side effects.